core/src/com/unciv/logic/city/managers/CityPopulationManager.kt

```diff
@@ -108,6 +108,12 @@ class CityPopulationManager : IsPartOfGameInfoSerialization {
         // We could simply test food==0 - but this way NullifiesStat(food) will still allow growth:
         if (city.getMatchingUniques(UniqueType.NullifiesGrowth).any())
             return
+        
+        // Hard block growth when using Avoid Growth, cap stored food
+        if (city.avoidGrowth) {
+            foodStored = foodNeededToGrow
+            return
+        }
```

Comment:
I thought this was discussed elsewhere and decided against. This goes beyond a simple bug fix and changes actual civ 5 gameplay, and thus either should get more discussion or be a unique

Reply:
Removed